modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/test/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelperTest.java

```diff
@@ -1158,6 +1159,60 @@ public void testValidationExpression() throws Exception {
 		Assert.assertFalse((boolean)ddmFormFieldPropertyChanges.get("valid"));
 	}
 
+	@Test
+	public void testValidationExpressionWithDifferentNumericFieldTypes()
+		throws Exception {
+
```

Comment:
@natocesarrego , there are already tests that validate if just a expression "2 != 2.0" is accepted, so do you think it is still necessary to check the status of the DDMFormField after the evaluation?

Reply:
Can you please give me more details about this comment, @mlan0? I'm not sure if I got your point, sorry. :grin:

Reply:
There is a test called `testNotEquals2` in `DDMExpressionImplTest.java`, that evaluates just the expression "2 != 2.0". The test `testValidationExpressionWithDifferentNumericFieldTypes` evaluates the expression, but also creates a DDMFormField to assert the error message and the valid attribute. 

So the question is, do you think its is better to only change the test(testNotEquals2) that evalutes only the expression?